api/lib/mno_enterprise/concerns/controllers/jpi/v1/teams_controller.rb

```diff
@@ -16,7 +16,7 @@ module MnoEnterprise::Concerns::Controllers::Jpi::V1::TeamsController
   # GET /mnoe/jpi/v1/organizations/:organization_id/teams
   def index
     authorize! :read, parent_organization
-    @teams = parent_organization.teams
+    @teams = MnoEnterprise::Team.where(organization_id: params[:organization_id])
```

Comment:
If I remember correctly, the reload was not working. That is why we thought it would be better do a proper load.